README.md

```diff
@@ -16,25 +16,33 @@ Version 3 moves away from a global setup approach to the use of an Intercom Clie
 
 This version of the gem is compatible with `Ruby 2.1` and above.
 
+
 ## Installation
 
+i
```

Comment:
This looks like a typo 🤔

Reply:
Agh I thought I got rid of that haha I'll just remove it here 😂